packages/xo-server/src/api/xostor.mjs

```diff
@@ -197,18 +197,22 @@ export const create = defer(async function (
 
     const boundInstallDependencies = installDependencies.bind(this)
     await asyncEach(hosts, host => boundInstallDependencies({ host }), { stopOnError: false })
+
+    const host = hosts[0]
+    const xapi = this.getXapi(host)
+
     const boundFormatDisks = formatDisks.bind(this)
     await asyncEach(
```

Comment:
You should use `asyncMapSettled` because `$defer()` cannot be called after the decorated function has finished, which can happen with `asyncEach` in case of rejection.

Reply:
The `stopOnError` option is set to `false`. So that shouldn't be a problem, right?

Reply:
My bad you're right!